guides/source/testing.md

```diff
@@ -843,12 +829,13 @@ users(:david)
 # this will return the property for david called id
 users(:david).id
 
-# one can also access methods available on the User class
+# methods available on the User class can also be accessed
```

Comment:
Nitpick, feel free to ignore, but "methods available to" seems more common in the guides:
```bash
$ ripgrep "methods available to"
guides/source/active_record_basics.md
329:models will have a number of helpful methods available to them. For example, you
activerecord/lib/active_record/schema.rb
37:        # Eval the given block. All methods available to the current connection
actionview/test/template/test_case_test.rb
202:    test "is able to make methods available to the view" do
railties/lib/rails/commands/console/irb_console.rb
13:      description "Gets helper methods available to ApplicationController."
```

```suggestion
# methods available to the User class can also be accessed
```